### ISSUE TITLE
Support permalinks in non-text files

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode-pull-request-github#2746]
I tried to get a permalink to an image but it gives you an error:

![image](https://user-images.githubusercontent.com/2193314/119284098-be5ac980-bbf3-11eb-9466-eec729b3a0ff.png)

**Comment by alexr00** [comment]
Looks like there isn't VS Code API to get the active editor if it isn't a text editor. In the mean time, you can use the permalink command on the editor tab to make such permalinks.

**Comment by Tyriar** [comment]
Blocked by https://github.com/microsoft/vscode/issues/133532

**Comment by TylerLeonhardt** [comment]
This worked for a `png` where we have a custom editor that will render the image, but it didn't work in this case:

![image](https://user-images.githubusercontent.com/2644648/214652687-6d2b1ad1-1626-447b-a458-e8e2eee64b85.png)

a `.icns` file (which is really just an image... but we don't render it I guess) that didn't load because it's an image trying to be rendered with the text editor.

I'll say "verification found" but I feel like maybe this is a new issue?

**Comment by alexr00** [comment]
Let's call it a new issue: https://github.com/microsoft/vscode-pull-request-github/issues/4468
